src/main/scala/org/constellation/domain/redownload/RedownloadService.scala

```diff
@@ -520,6 +549,9 @@ object RedownloadService {
   type SnapshotsAtHeight = Map[Long, String] // height -> hash
   type SnapshotProposalsAtHeight = Map[Long, SnapshotProposal]
   type PeersProposals = Map[Id, SnapshotProposalsAtHeight]
+  type PeersCache = Map[Id, MajorityHeight]
   type SnapshotInfoSerialized = Array[Byte]
   type SnapshotSerialized = Array[Byte]
+
+  case class LatestMajorityHeight(low: Long, high: Long)
```

Comment:
`lowest`, `highest` or `min` `max` imo.

`low` or `high` doesn't mean `lowest` and `highest`

Reply:
done